packages/fpdart/README.md

```diff
@@ -346,9 +346,17 @@ String goShoppingDo() => Option.Do(
     );
 ```
 
+> **Warning** The **Do notation** has some limitations that need to be considered. Do **not** use it when:
```

Comment:
I would move this quote below the other quote (i.e. `> **Note**: ...`)

Reply:
I would also add the warning emoji and rephrase:
```suggestion
> ⚠️ **Warning**: Pay attention to avoid the following mistakes when using the **Do notation**:
```

I would then change the list in the form "Do not throw inside ..." (what **not to do**)

Could you also add a code snippet with examples of mistakes for each point in the list?